app/templates/_common/comment.html.twig

```diff
@@ -17,7 +17,11 @@
                 <div class="row">
                     <h3>
                         <div class="avatar pull-right">
-                            <img src="{{ gravatar(comment.getGravatarHash) }}">
+                            {% if comment.getUserDisplayName %}
+                                <img src="{{ gravatar(comment.getGravatarHash) }}">
+                            {%  else %}
+                                <img src="https://secure.gravatar.com/avatar/294de3557d9d00b3d2d8a1e6aab028cf?d=mm&s=40">
```

Comment:
Shouldn't we use the way that is described at https://en.gravatar.com/site/implement/images/ under "Default Image"? 

Instead of providing a hash to an image that is nowhere documented where it leads to? what email address is that? who's in charge of that account?

Reply:
That's the md5 hash of "anonymous"

Reply:
Updated to all 0s and default forced to mystery person